src/com/android/volley/Request.java

```diff
@@ -667,4 +667,8 @@ public boolean shouldCacheInstantly() {
     public void onParseOOM(OutOfMemoryError e) {
         // no-op
     }
+
+    public void expireCache() {
+        // no - op
```

Comment:
this exists because the base class Request has it so you can use it without casting, but only things extending AirbnbRequest need it.